core/schema/container.graphqls

```diff
@@ -86,7 +86,7 @@ type Container {
   fs: Directory! @deprecated(reason: "Replaced by `rootfs`.")
 
   "Initializes this container from this DirectoryID."
-  withRootfs(id: DirectoryID!): Container!
+  withRootfs(directory: DirectoryID!): Container!
```

Comment:
I don't disagree with this name, but want to flag that other APIs like `withMountedDirectory` call this `source`. I think `directory` is OK here case because there's nothing to disambiguate it against, whereas `withMountedDirectory` also has a "target" path. But then `withMountedDirectory` just calls it `path` anyway so there's no symmetry like source/target. :man_facepalming:

Just raising to see what you think. Would it make sense to rename the others to `directory` too? :thinking: (Not necessarily as part of this PR; approving as-is.)

Reply:
Yeah, I initially had `source` in the issue but then changed to directory to mirror `withDirectory(path: String!, directory: DirectoryID!)`.

So in mounts `source` is used, in builds `context` is used and `directory` outside of that.

Reply:
I'm good with renaming these for consistency though in a follow-up.